itl/command-plugins.conf

```diff
@@ -549,6 +549,195 @@ object CheckCommand "http" {
+		}
+		"--sni" = {
+			set_if = "$curl_sni$"
+			description = "Enable SSL/TLS hostname extension support (SNI). Default if TLS version > 1.0"
+		}
+		"-C" = {
+			value = "$curl_certificate_valid_days_min_warning$,$curl_certificate_valid_days_min_critical$"
+			description = "Minimum number of days a certificate has to be valid."
+		}
+		"--continue-after-certificate" = {
+			value = "$curl_continue_after_certificate$"
+			description = "Allows the HTTP check to continue after performing the certificate check. Does nothing unless -C is used."
+		}
+		"-J" = {
+			value = "$curl_client_certificate_file$"
+			description = "Name of file that contains the client certificate (PEM format) to be used in establishing the SSL session"
+		}
+		"-K" = {
+			value = "$curl_client_certificate_key_file$"
+			description = "Name of file containing the private key (PEM format) matching the client certificate"
+		}
+		"--ca-cert" = {
+			value = "$curl_ca_cert_file$"
+			description = "CA certificate file to verify peer against"
+		}
+		"-D" = {
+			set_if = "$curl_verify_peer_cert$"
+			description = "Verify the peer's SSL certificate and hostname"
+		}
+		"-e" = {
+			value = "$curl_expext_string$"
+			description = "Comma-delimited list of strings, at least one of them is expected in the first (status) line of the server response (default: HTTP/), If specified skips all other status line logic (ex: 3xx, 4xx, 5xx processing)"
+		}
+		"-d" = {
+			value = "$curl_expect_header_string$"
+			description = "String to expect in the response headers"
+		}
+		"-s" = {
+			value = "$curl_expect_content_string$"
+			description = "String to expect in the content"
+		}
+		"-u" = {
+			value = "$curl_url$"
+			description = "URL to GET or POST (default: /)"
+		}
+		"-P" = {
+			value = "$curl_post_data$"
+			description = "URL encoded http POST data"
+		}
+		"-j" = {
+			value = "$curl_http_method$"
+			description = "Set HTTP method (for example: HEAD, OPTIONS, TRACE, PUT, DELETE, CONNECT)"
+		}
+		"-N" = {
+			value = "$curl_no_body$"
+			description = "Don't wait for document body: stop reading after headers. (Note that this still does an HTTP GET or POST, not a HEAD.)"
+		}
+		"-M" = {
+			value = "$curl_max_age$"
+			description = "Warn if document is more than SECONDS old. the number can also be of the form '10m' for minutes, '10h' for hours, or '10d' for days."
+		}
+		"-T" = {
+			value = "$curl_content_type$"
+			description = "specify Content-Type header media type when POSTing"
+		}
+		"-l" = {
+			value = "$curl_linespan$"
+			description = "Allow regex to span newlines (must precede -r or -R)"
+		}
+		"-r" = {
+			value = "$curl_ereg$"
+			description = "Search page for regex STRING"
+		}
+		"-R" = {
+			value = "$curl_eregi$"
+			description = "Search page for case-insensitive regex STRING"
+		}
+		"--invert-regex" = {
+			set_if = "$curl_invert_regex$"
+			description = "When using regex, return CRITICAL if found, OK if not"
+		}
```

Comment:
I am really sorry, but I must have missed it the last two times: `--regex-state=STATE` is missing.

Reply:
Thanks for mentioning it ( ... [hach](https://github.com/monitoring-plugins/monitoring-plugins/pull/2014) )